.github/workflows/build.yml

```diff
@@ -30,6 +31,11 @@ jobs:
         go-version: ['1.24']
       fail-fast: true
     steps:
+      - name: Check signed commits in PR
+        if: github.event_name == 'pull_request'
+        uses: 1Password/check-signed-commits-action@ed2885f3ed2577a4f5d3c3fe895432a557d23d52 #v1.2.0
+        with:
+          comment: One or more commits in your pull request are not signed. The Notation project requires all commits to be signed. Please sign your commits by following the [guide](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits). Thank you!
```

Comment:
```suggestion
          comment: One or more commits in your pull request are not signed. The Notary Project requires all commits to be signed. Please sign your commits by following the [guide](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits). Thank you!
```

Reply:
The project brand is `Notary Project`. It would be better to reference the project brand.